docs/chapters/04_features.md

```diff
@@ -446,6 +463,134 @@ mutation {
 
 > [!NOTE]  Remember that, in case you want to subscribe to a read model that is restricted to a specific set of roles, you must send the **access token** retrieved upon sign-in. Check ["Authorizing operations"](#authorizing-operations) to know how to do this.
 
+### Reading events
+
+You can also fetch events directly if you need. To do so, there are two kind of queries that have the following structure:
+```graphql
+query {
+  eventsByEntity(entity: <name of entity>, entityID: "<id of the entity>") {
+    selection_field_list
+  }
+}
+
+query {
+  eventsByType(type: <name of event>) {
+    selection_field_list
+  }
+}
+```
+Where:
+- _<name of your entity>_ is the name of the class corresponding to the entity whose events you want to retrieve.
```

Comment:
For some reason, all the words between < and > are not being rendered in the final ReadMe. I guess it is because they are interpreted as HTML tags and they are stripped. May be using the special symbols &gt and the like would work (check how it was done in the other places 😉 )